charts/rapid-response/values.yaml

```diff
@@ -6,6 +6,40 @@ global:
   image:
     pullSecrets: []
     pullPolicy: IfNotPresent
+  ssl:
+    ca:
+      # For outbound connections (secure backend, proxy,...)
+      # A PEM-encoded x509 certificate.  This can also be a bundle with multiple certificates.
+      # See below in the rapidResponse.ssl.ca.cert key for an example
+      cert:
```

Comment:
This should be `certs: []` to keep the same strcuture we have in the other components.